src/test/scala/com/microsoft/hyperspace/index/IndexLogManagerImplTest.scala

```diff
@@ -166,6 +166,27 @@ class IndexLogManagerImplTest
     assert(actualActiveVersions.equals(Seq(3, 1)))
   }
 
+  test("testGetLatestStableLog shouldn't return irrelevant previous log.") {
+    val path = new Path(testRoot, UUID.randomUUID().toString)
+    val fs = path.getFileSystem(new Configuration)
+
+    FileUtils.createFile(
+      fs,
+      new Path(path, s"$HYPERSPACE_LOG/8"),
+      JsonUtils.toJson(getEntry("ACTIVE")))
+    FileUtils.createFile(
+      fs,
+      new Path(path, s"$HYPERSPACE_LOG/10"),
+      JsonUtils.toJson(getEntry("VACUUMING")))
```

Comment:
Don't you need another assert here so that we check both `VACUUMING` and `CREATING`?